### PR TITLE
Feature/#161 outdated tool

### PR DESCRIPTION
**Pull request by rs-michael**
This PR introduces a script that allows the developer to quickly check for outdated tools and images that are in use.

The scripts fetches the latest version for a given tool, extracts the currently used version in the docker file and if possible, any locally installed tools. When run locally it is able to print outdated versions and prints a simple error message to stderr. This script has been added as part of the azure pipeline but can also be added as part of the pre-commit hook configuration.
image
image

### CODE REVIEW

**Comment by rs-michael** [comment]
The build will succeed all previous tasks/scripts, but will fail with the introduction of the new version checking script.

**Review by mgonzales-rs — APPROVED** [review]
LGTM
only suggestion will try to move this file to /scripts

File: scan.py
Comment (vitali-pozniak) [inline scan.py:1]: Hi, @rs-michael , I agree with @mgonzales-rs , please the file to /scripts.

**Comment by rs-michael** [comment]
> LGTM only suggestion will try to move this file to /scripts

done

**Review by vitali-pozniak — APPROVED** [review]
lgtm

**Pull request merged**